custom-elements/connected-callbacks-html-fragment-parsing.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Custom Elements: connectedCallback not called during HTML fragment parsing</title>
+<meta name="author" title="Rob Buis" href="mailto:rbuis@igalia.com">
+<meta name="assert" content="connectedCallback must not be called during HTML fragment parsing">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/parsing.html#create-an-element-for-the-token">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/custom-elements-helpers.js"></script>
+</head>
+<body>
+<div id="log"></div>
+<script>
+
+var calls = 0;
+class Parenter extends HTMLElement {
```

Comment:
This class should be re-defined for each document type.
e.g. you should be declaring these classes inside promise_test itself.